blocks/carousel/block.php

```diff
@@ -5,6 +5,8 @@
 $slides_per_group = get_field('cz_carousel_slides_per_group') ? (int) get_field('cz_carousel_slides_per_group') : $slides_per_view;
 $space_between = get_field('cz_carousel_space_between') ? (int) get_field('cz_carousel_space_between') :30;
 $loop = get_field('cz_carousel_auto_height') !== null ? get_field('cz_carousel_auto_height') : false;
+$pagination = get_field('cz_carousel_pagination') !== null ? get_field('cz_carousel_pagination') : false;
+$pagination_active_color = get_field('cz_carousel_pagination_active_color');
```

Comment:
@x3blondie, I believe you made this change based on a previous comment I deleted. Let's remove this color field in favor of just setting `:root { --swiper-theme-color: }` and otherwise manually customizing CSS however we like for each theme. There are so many swiper color possibilities, I wouldn't want to attempt to support them all as block options.